### PR TITLE
js if

### PR DESCRIPTION
**Pull request by louiewhitz**
Screen Shot 2022-08-31 at 12 35 09 PM
Screen Shot 2022-08-31 at 12 32 02 PM
Screen Shot 2022-08-31 at 12 31 50 PM

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  }
+  return false;
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'j' || string[0] === 'J') {
+    return true;
+  }
+  return false;
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js]: To be fair, this should only return `true` if the string begins with a _capital_ `'J'`:

---

image

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else
+  if ((pH <= 7) && (pH >= 0)) {
+    return 'acid';
+  } else
+  if (pH <= 14 && pH >= 7) {
+    return 'base';
+  } else {
```
Comment (TimelikeClosure) [inline javascript-if/main.js]: For clarity, you may want to remove the "equals to `7`" part of the acid and base comparisons.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-if/main.js (hunk 1 above)
Comment (louiewhitz) [reply javascript-if/main.js]: Definitely misread, thank you!

File: javascript-if/main.js (hunk 2 above)
Comment (louiewhitz) [reply javascript-if/main.js]: I definitely agree :)

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**